### PR TITLE
[gardening] Remove unused variable loweredMT

### PR DESCRIPTION
**Pull request by practicalswift**

### CODE REVIEW

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

File: lib/SIL/SILVerifier.cpp [hunk 1]
```diff
@@ -1664,7 +1664,7 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     }
 
     // Metatypes preserve their instance type through lowering.
-    if (auto loweredMT = loweredType.getAs<MetatypeType>()) {
+    if (loweredType.getAs<MetatypeType>()) {
```
Comment (slavapestov) [inline lib/SIL/SILVerifier.cpp]: This should be `.is<MetatypeType>()` since the result is not needed

**Comment by practicalswift** [comment]
@slavapestov Good point. Fixed, squashed and pushed. Thanks!

**Comment by practicalswift** [comment]
@swift-ci please smoke test

**Comment by practicalswift** [comment]
@swift-ci please test

**Pull request merged**